.github/release-drafter.yml

```diff
@@ -0,0 +1,17 @@
+name-template: 'v$RESOLVED_VERSION'
+tag-template: 'v$RESOLVED_VERSION'
+categories:
+  - title: 🚀 Features
+    label: 'enhancement'
+  - title: 🐛 Bug Fixes
+    label: 'bug'
+  - title: 🛠 Internal Changes
+    label: 'internal'
```

Comment:
we can change this to whatever makes sense.

Reply:
Nit Suggestion : Workflow Changes